README.md

```diff
@@ -72,10 +72,120 @@ Finally, add the bundle as a script tag in your page, and initialize the player
   player.play();
 </script>
 ```
+</br>
+</br>
 
 ## Documentation
 
-- **[Configuration](docs/configuration.md)**
+Share plugin enables to share the current content or some other content related url to the social media.
+
+Share plugin also enables customizing the sharing attributes. 
+
+If share plugin is enabled  share icon will be added to the player's top right corner configurable using the uiComponent [area](https://github.com/kaltura/playkit-js-ui/blob/master/docs/ui-components.md#general) attribute.
+
+```
+"uiComponent": {
+          "label": 'shareButtonComponent',
+          "presets": ['Playback', 'Live'],
+          "area": 'TopBarRightControls'
+},
+```
+
+
+
+#### Sharing Capabilities
+
+* Share to a social networks
+* Email a link to the video
+* Copy a code which enables the video to be embedded
+
+#### UI Examples
+
+![example](./images/shareIcon.png)
+
+![example](./images/shareScreen.png)
+
+#### Configuration attributes:
+#### shareUrl
+
+* The shareUrl to use if we want to force such and not using the default share url (using this option will override the definition you have given in `shareOptions`)
```

Comment:
```suggestion
* The shareUrl to use if we want to a custom url instead of using the default share url (using this option will override the definition you have given in `shareOptions`)
```